src/Filament/Resources/LinkCheckResource.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace Portable\FilaCms\Filament\Resources;
+
+use Filament\Forms\Form;
+use Filament\Tables\Columns\TextColumn;
+use Filament\Tables\Table;
+use Illuminate\Database\Eloquent\Builder;
+use Portable\FilaCms\Filament\Resources\LinkCheckResource\Pages;
+use Portable\FilaCms\Filament\Traits\IsProtectedResource;
+use Portable\FilaCms\Models\LinkCheck;
+
+class LinkCheckResource extends AbstractResource
+{
+    use IsProtectedResource;
+
+    protected static ?string $model = LinkCheck::class;
+
+    protected static ?string $navigationIcon = 'heroicon-o-link';
+    protected static ?string $navigationGroup = 'System';
+
+    protected static ?string $navigationBadgeTooltip = 'The number of broken links';
+
+    public static function form(Form $form): Form
+    {
+        $model = $form->model;
+
+        return $form
+            ->schema([
+
+            ]);
+    }
+
+    public static function getNavigationBadge(): ?string
+    {
+        $batch = (new (static::getModel()))->latestBatch();
+
+        $count = static::getModel()::where('batch_id', $batch)
+            ->whereNotBetween('status_code', [200, 299])
+            ->count();
+
+        return $count;
+    }
+
+    public static function getNavigationBadgeColor(): ?string
+    {
+        $batch = (new (static::getModel()))->latestBatch();
+
+        $count = static::getModel()::where('batch_id', $batch)
+            ->whereNotBetween('status_code', [200, 299])
+            ->count();
+
+        return $count === 0 ? 'success' : 'warning';
```

Comment:
You should create a method to get the count. If the count is not saved to a class property then do DB calls to get the count.